src/02-crm/tags.apib

```diff
@@ -0,0 +1,29 @@
+## Tags [/tags]
+
+### tags.list [GET /tags.list]
+
+Get a list of tags.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + name
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+            + Default
+                + (object)
+                    + field: `name`
+                    + order: `asc`
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + name: `Client` (string)
```

Comment:
Can we lowercase all tags?

Reply:
They aren't lowercased in the UI, so would this mean that when we move to using the new API internally, all tags will have to be lowercased? Not sure if this is a behavioural change we want?

Reply:
We want tags to become unique as well, I prefer case-insensitive tags.